packages/mux-player/src/index.ts

```diff
@@ -172,6 +173,29 @@ class MuxPlayerInternal {
     }
   }
 
+  _setUpLiveFirstPlay(el: MuxPlayerElement) {
+    if (el.video?.hls) {
```

Comment:
`hls` won't exist if there is not a `playback-id`/`src` set, which prompts an `initialize()` in playback-core.

Reply:
yeah, this should probably move into playback-core

Reply:
event types are those that are live but playback should start from the beginning or from EXT-X-START? In which case, we wouldn't want to do this seek. I'll have to see if and how hls.js exposes this info.

For reference, VHS doesn't currently support EXT-X-START or starting at the beginning of a live playlist.